frontend/public/components/modals/add-secret-to-workload.tsx

```diff
@@ -167,30 +169,33 @@ export class AddSecretToWorkloadModal extends React.Component<
   };
 
   render() {
+    const { t } = this.props;
     const { secretName } = this.props;
     const { addAs, workloadOptions, selectedWorkloadUID } = this.state;
     const addAsEnvironment = addAs === 'environment';
     const addAsVolume = addAs === 'volume';
-    const selectWorkloadPlaceholder = 'Select a workload';
+    const selectWorkloadPlaceholder = t('modal~Select a workload');
 
     return (
       <form
         onSubmit={this.submit}
         name="co-add-secret-to-workload"
         className="co-add-secret-to-workload modal-content"
       >
-        <ModalTitle>Add Secret to Workload</ModalTitle>
+        <ModalTitle>{t('modal~Add Secret to Workload')}</ModalTitle>
```

Comment:
As part of this effort, we're updating the capitalization to match the conventions here: http://openshift.github.io/openshift-origin-design/conventions/documentation/capitalization.html

```suggestion
        <ModalTitle>{t('modal~Add Secret to workload')}</ModalTitle>
```

Reply:
will change it